.travis.yml

```diff
@@ -14,12 +14,7 @@ php:
     - hhvm
 
 env:
-    global:
-        - TEST_COMMAND="composer test"
-    matrix:
-        - SYMFONY_VERSION=3.2.*
-        - SYMFONY_VERSION=3.1.*
-        - SYMFONY_VERSION=2.8.*
+    - SYMFONY_VERSION=2.8.* TEST_COMMAND="composer test"
```

Comment:
See https://github.com/Incenteev/IncenteevTranslationCheckerBundle/blob/169c90a9bfc9c7475883521a0edfee30c1c141a0/.travis.yml for an example of such setup (with too much supported PHP versions)